modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -50,6 +50,122 @@ ifndef::openshift-origin[]
 link:https://cloud.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {cloud-redhat-com} site and save it to a `.json` file.
 endif::[]
 
+. Generate the base64-encoded user name and password or token for your mirror
+registry:
++
+[source,terminal]
+----
+$ echo -n '<user_name>:<password>' | base64 -w0 <1>
+BGVtbYk3ZHAtqXs=
+----
+<1> For `<user_name>` and `<password>`, specify the user name and password that
+you configured for your registry.
+
+ifndef::openshift-origin[]
+. Make a copy of your pull secret in JSON format:
++
+[source,terminal]
+----
+$ cat ./pull-secret.text | jq .  > <path>/<pull-secret-file><1>
+----
+<1> Specify the path to the folder to store the pull secret in and a name for
+the JSON file that you create.
++
+The contents of the file resemble the following example:
++
+[source,json]
+----
+{
+  "auths": {
+    "cloud.openshift.com": {
+      "auth": "b3BlbnNo...",
+      "email": "you@example.com"
+    },
+    "quay.io": {
+      "auth": "b3BlbnNo...",
+      "email": "you@example.com"
+    },
+    "registry.connect.redhat.com": {
+      "auth": "NTE3Njg5Nj...",
+      "email": "you@example.com"
+    },
+    "registry.redhat.io": {
+      "auth": "NTE3Njg5Nj...",
+      "email": "you@example.com"
+    }
+  }
+}
+----
+endif::[]
+
+ifndef::openshift-origin[]
+. Edit the new
+endif::[]
+ifdef::openshift-origin[]
+. Create a `.json`
```

Comment:
@vrutkovs, does this pull secret file seem reasonable for OKD?

Reply:
yeah, lgtm